releasenotes/notes/agent-log-relative-file-fd9022ec8b80ec0d.yaml

```diff
@@ -0,0 +1,4 @@
+---
+enhancements:
+  - |
+    The logs now contains the relative file path (including the package) instead of only the filename
```

Comment:
to avoid confusion with the Logs feature, may be worth writing `The Agent logs now contain [...]`